server/shared-models/src/main/scala/com/prisma/shared/models/Project.scala

```diff
@@ -20,14 +20,18 @@ case class Project(
   val serverSideSubscriptionFunctions = functions.collect { case x: ServerSideSubscriptionFunction => x }
 
   val dbName: String = manifestation match {
-    case ProjectManifestation(Some(_), Some(schema)) => schema // Postgres Passive
-    case ProjectManifestation(Some(database), None)  => database // Mongo + MySQL Passive
-    case ProjectManifestation(None, Some(_))         => sys.error("The configloader should have rejected this.") // Invalid
-    case ProjectManifestation(None, None)            => id // All active
+    case ProjectManifestation(Some(_), Some(schema), "postgres")                  => schema
+    case ProjectManifestation(Some(_), Some(schema), _)                           => sys.error("The configloader should have rejected this.")
```

Comment:
Can you add explanations why those cases that error are invalid? I can't remember anymore :D

Reply:
Done